services/alerta/service.go

```diff
@@ -22,6 +24,7 @@ const (
 	defaultResource    = "{{ .Name }}"
 	defaultEvent       = "{{ .ID }}"
 	defaultGroup       = "{{ .Group }}"
+	defaultTimeout     = toml.Duration(86400 * time.Second)
```

Comment:
This reads a little better as `time.Duration(24*time.Hour)`. Also once the duration has been parsed from the config its best to use `time.Duration` so that normal time operations work.